AppInfrastructure/Public/Dobby/IDobbyProxy.h

```diff
@@ -28,8 +28,6 @@
 
 #if defined(RDK)
 #  include <json/json.h>
```

Comment:
Why is this still here if we've removed the jsoncpp as a dependency?

Reply:
Fixed